src/foremast/pipeline/construct_pipeline_block.py

```diff
@@ -142,6 +142,12 @@ def construct_pipeline_block(env='',
     hc_grace_period = data['asg'].get('hc_grace_period')
     app_grace_period = data['asg'].get('app_grace_period')
     grace_period = hc_grace_period + app_grace_period
+    
+    ssh_keypair = data['asg'].get('ssh_keypair', None)
+    LOG.info('SSH keypair ({0}) used'.format(ssh_keypair))
+    if not ssh_keypair:
+        ssh_keypair = '{0}_{1}_default'.format(env, region)
```

Comment:
The default should be defined in `configs.json`. It may avoid some of this logic.
Although, maybe we want the ability to *not* set a ssh key, for teams that do not want ssh access at all.